admin/src/pages/HomePage/components/NavigationItemForm/index.tsx

```diff
@@ -743,34 +755,28 @@ export const NavigationItemForm: React.FC<NavigationItemFormProps> = ({
                             : undefined
                         }
                       >
-                        <Combobox
+                        <SingleSelect
```

Comment:
I would change it to `ControllableCombobox` as there were cases when someone got many related types